server/controllers/emptyBuildings/post.js

```diff
@@ -0,0 +1,35 @@
+const { promisify } = require('util');
+const { reportBuilding } = require('../../models/queries/emptyBuildings');
+const { schemaBuildings } = require('../../validation');
+
+const postEmptyBuilding = async (req, res, next) => {
+  try {
+    const newBuild = await schemaBuildings.validate(req.body, {
+      abortEarly: false,
+    });
+    if (req.files) {
+      const file = req.files.filename;
+      const fileName = `${Date.now()}${file.name}`;
+      newBuild.thumbnail = fileName;
+      const move = promisify(file.mv);
+      move(`./server/uploads/${fileName}`)
+        .then(filename => res.send({ statusCode: 201, data: filename }))
```

Comment:
Right now, I think both `res.send` here and `res.status(201).send` below can be called for a single request. Is that expected?